src/Chat.tsx

```diff
@@ -76,9 +76,9 @@ export class Chat extends React.Component<ChatProps, {}> {
         konsole.log("BotChat.Chat props", props);
 
         if (props.pastHistory) {
```

Comment:
there's still a "pastHistory" on this line. You may want to do a global search, or use VSCode's "rename" feature.